common/test/views/support/cleaner/AffiliateLinksCleanerTest.scala

```diff
@@ -1,8 +1,10 @@
 package views.support.cleaner
 import conf.Configuration
+import conf.switches.Switches.ServerSideExperiments
 import org.joda.time.DateTime
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import play.api.test.FakeRequest
 import views.support.AffiliateLinksCleaner._
 
 class AffiliateLinksCleanerTest extends AnyFlatSpec with Matchers {
```

Comment:
I applaud the fakery going on in this file, but I think it would be simpler to achieve using the [`FakeRequests`](https://github.com/guardian/frontend/blob/main/common/test/implicits/FakeRequests.scala) trait. This way, you don't need to explicitly pass in the `request` to every call to `shouldAddAffiliateLinks`.

```suggestion
import implicits.FakeRequests

class AffiliateLinksCleanerTest extends AnyFlatSpec with Matchers with FakeRequests {
```

Reply:
Discussed with Simon and this doesn't seem to resolve the issue so leaving for now